test/config-next/cert-checker.json

```diff
@@ -10,12 +10,14 @@
 		"badResultsOnly": true,
 		"checkPeriod": "72h",
 		"acceptableValidityDurations": [
-			"7776000s"
+			"7776000s",
+			"160h"
 		],
 		"lintConfig": "test/config-next/zlint.toml",
 		"ignoredLints": [
-			"w_subject_common_name_included",
 			"w_ext_subject_key_identifier_missing_sub_cert",
+			"w_subject_common_name_included",
+			"e_dnsname_not_valid_tld",
```

Comment:
This needs a comment explaining why we ignore this lint. Unfortunately json doesn't allow comments...

We ignore this lint because we check TLDs against the PSL, and it's faster to update our copy of the PSL than it is to wait for zlint to update it. Since this ignore depends on code, we should probably implement the ignore in code - that is, automatically add this to the list of ignored lints when we configure zlint. Then we can simply add a comment there.

Also, the list of ignored lints should really be the same between ca.json and cert-checker.json.

Lastly: can you remind me why some lints are ignored here in the JSON config vs zlint.toml? Might require some git spelunking.

Reply:
Good catch on the CA lints - thanks! It turns out there were a lot of other divergences between the test and prod configs for the CA's cert profiles.

My idea with this PR was just to reduce those divergences between test and deployed prod configs, in general. It looks like prod doesn't currently use any kind of `lintConfig` structure at all.